README.md

```diff
@@ -47,7 +47,7 @@ It is a simple web app built with React, to demonstrate a project setup involvin
 ## 💻 Getting Started <a name="getting-started"></a>
 
 To get a local copy up and running, follow these steps.
-This app uses API from its back-end counterpart [front-end](https://github.com/adamilare/hello-react-back-end)
+This app uses API from its back-end counterpart [front-end](https://github.com/adamilare/hello-rails-back-end)
```

Comment:
- [Optional ] The above link redirect to the `dev` branch Kindly make sure that you redirect the the `feat-server `branch.